### PR TITLE
feat(new-ia): virtual adoption page

### PR DESCRIPTION
**Pull request by imklau**
## Description 📝

This PR adds a new page `/adopcja-writualna` based on the `/adopcja`.
## Related Issues 🔗

JIRA issue: [FSU-377](https://klau.atlassian.net/browse/FSU-377)

## Preview Links 🔍

- [/adopcja-wirtualna](https://sterczaceuszy-git-fsu-377-virtual-adoption-sterczace-uszy.vercel.app/adopcja-wirtualna)

### CODE REVIEW

**Review by agnieszkastanczuk — APPROVED** [review]

**Review by agnieszkastanczuk — APPROVED** [review]

File: packages/frontend/app/templates/VirtualAdoption/components/BasicInfo/BasicInfo.styled.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  grid-auto-flow: rows;
+  gap: 60px;
+`
+
+export const FeatureItem = styled.div`
+  max-width: 760px;
+`
+
+export const AdditionalNote = styled(Typography)`
+  max-width: 760px;
```
Comment (paulgrym) [inline packages/frontend/app/templates/VirtualAdoption/components/BasicInfo/BasicInfo.styled.js:51]: I think it is 506px in the new design

File: packages/frontend/app/templates/VirtualAdoption/components/BasicInfo/BasicInfo.styled.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+export const List = styled.div`
+  margin-top: 100px;
+  display: grid;
+  grid-auto-flow: rows;
+  gap: 60px;
+`
+
+export const FeatureItem = styled.div`
+  max-width: 760px;
```
Comment (paulgrym) [inline packages/frontend/app/templates/VirtualAdoption/components/BasicInfo/BasicInfo.styled.js:47]: 696px in the new design

**Review by paulgrym — COMMENTED** [review]
There is one more section in tablet and mobile view in Figma, but it looks like it is a mistake
image
Good job!

File: packages/frontend/app/templates/VirtualAdoption/VirtualAdoption.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      <BasicInfo />
+      <Steps />
+      <ChooseYourVirtualPet data={animalsData.items} />
+      <RecurringPayment />
+      <PaymentConfirmationForm data={animals.items} />
+      <CtaBanner
+        heading="Masz pytania lub wątpliwości?"
+        buttonText="Skontaktuj się z nami"
+        buttonHref="/kontakt"
+      />
```
Comment (paulgrym) [inline packages/frontend/app/templates/VirtualAdoption/VirtualAdoption.jsx:44]: Banner is aligned to the left on tablet and mobile in the new design

**Review by AmandaKr — APPROVED** [review]

File: packages/frontend/app/templates/VirtualAdoption/components/BasicInfo/BasicInfo.styled.js (hunk 1 above)
Comment (imklau) [inline packages/frontend/app/templates/VirtualAdoption/components/BasicInfo/BasicInfo.styled.js:51]:
```suggestion
  max-width: 506px;
```

File: packages/frontend/app/templates/VirtualAdoption/components/BasicInfo/BasicInfo.styled.js (hunk 2 above)
Comment (imklau) [inline packages/frontend/app/templates/VirtualAdoption/components/BasicInfo/BasicInfo.styled.js:47]:
```suggestion
  max-width: 696px;
```

**Pull request merged**